cluster/src/main/java/com/hartwig/pipeline/calling/structural/gripss/Gripss.java

```diff
@@ -27,12 +27,13 @@
 import com.hartwig.pipeline.reruns.PersistedDataset;
 import com.hartwig.pipeline.reruns.PersistedLocations;
 import com.hartwig.pipeline.resource.ResourceFiles;
+import com.hartwig.pipeline.stages.Namespace;
 import com.hartwig.pipeline.stages.Stage;
 import com.hartwig.pipeline.storage.GoogleStorageLocation;
 import com.hartwig.pipeline.storage.RuntimeBucket;
 import com.hartwig.pipeline.tools.Versions;
 
-public class Gripss implements Stage<GripssOutput, SomaticRunMetadata> {
+public abstract class Gripss implements Stage<GripssOutput, SomaticRunMetadata> {
```

Comment:
nor here

Reply:
I changed this to make it easier to distinguish between leaf nodes on the `Stage` class structure and those in the middle of the structure for the purposes of writing a useful test to check for classes missing the new `Namespace` annotation.

I had the same objection as you but then thought about whether instantiating this class directly would make sense and decided it wouldn't. This makes sense to me but maybe I'm missing something here. Let's discuss further so I can understand where you're coming from.